ViAn/GUI/projectwidget.cpp

```diff
@@ -54,14 +55,16 @@ void ProjectWidget::create_default_tree() {
 void ProjectWidget::add_video() {
     if (m_proj == nullptr)  return; // TODO: HANDLE CASE
 
-    QString video_path = QFileDialog().getOpenFileName(this, tr("Add video"), m_proj->getDir_videos().c_str());
-    int index = video_path.lastIndexOf('/') + 1;
-    QString vid_name = video_path.right(video_path.length() - index);
+    QStringList video_paths = QFileDialog().getOpenFileNames(this, tr("Add video"), m_proj->getDir_videos().c_str());
+    for (auto video_path : video_paths){
+        int index = video_path.lastIndexOf('/') + 1;
```

Comment:
Might want to check if the videopath is empty. now if you clicked add video and then closes the popup window you will get an empty string which will create an empty item in the tree.